assets/chezmoi.io/docs/reference/target-types.md

```diff
@@ -113,26 +113,40 @@ The default script interpreters are:
 | `.ps1`    | `powershell` | `-NoLogo` |
 | `.rb`     | `ruby`       | *none*    |
 
-Script interpreters can be added or overridden with the
-`interpreters.`*extension* section in the configuration file.
-
-!!! note
-
-    The leading `.` is dropped from *extension*, for example to specify the
-    interpreter for `.pl` files you configure `interpreters.pl`.
+Script interpreters can be added or overridden by adding the corresponding
+extension (without the leading dot) as a key under the `interpreters`
+section of the configuration file.
```

Comment:
I think that the `!!!note` is still useful, or something like it, because it makes it clear that the key for `.pl` is `pl`.

There are other changes that I would consider to show this, like the table:

```markdown
| Extension | Key   | Command      | Arguments |
| --------- | ----- | ------------ | --------- |
| `.pl`     | `pl`  | `perl`       | *none*    |
| `.py`     | `py`  | `python3`    | *none*    |
| `.ps1`    | `ps1` | `powershell` | `-NoLogo` |
| `.rb`     | `rb`  | `ruby`       | *none*    |
```

But that’s a bit more change than I would make without @twpayne agreeing. (Also, reading above, the bit about `%PATH%` should be `` in your `%PATH` or fully specified ``, but that’s *well* outside of the changes you’re contributing.

Reply:
Re-added the note.